.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+- package-ecosystem: gradle
+  directory: /
+  schedule:
+    interval: monthly
+- package-ecosystem: maven
```

Comment:
ok, you likely only need gradle. I said gradle/maven because gradle pulls artifacts from maven central, so I wasn't sure if github would combine them or whatever. Doesn't hurt to keep around I guess, I imagine it'll just silently fail.

Reply:
if it blows, I'll yeet it.